app/src/main/res/values/strings.xml

```diff
@@ -216,57 +216,57 @@
     <string name="dns_act_log">Logs</string>
 
     <string name="univ_firewall_dialog_title">Delete IP Rule?</string>
-    <string name="univ_firewall_dialog_message">Deleting this rule would allow any app to connect to this IP. Do you want to proceed?</string>
-    <string name="univ_firewall_no_ip_message">No IPs found. Tap on entries in the Network Log screen to block individual IP addresses.</string>
+    <string name="univ_firewall_dialog_message">Continue deleting this rule (allowing any app to connect to this IP)?</string>
+    <string name="univ_firewall_no_ip_message">No IPs found. Tap on entries in the \"Network Log\" screen to block individual IP addresses.</string>
 
     <string name="doh_custom_url_remove_dialog_title">Delete DoH endpoint?</string>
-    <string name="doh_custom_url_remove_dialog_message">Delete the DNS over Https endpoint from saved configuration.</string>
+    <string name="doh_custom_url_remove_dialog_message">Delete the DNS-over-HTTPS endpoint from saved configuration.</string>
     <string name="doh_custom_url_remove_success">Endpoint deleted.</string>
 
     <string name="dns_crypt_custom_url_remove_dialog_title">Delete DNSCrypt resolver?</string>
     <string name="dns_crypt_url_remove_dialog_message">Delete the DNSCrypt resolver from saved configuration.</string>
     <string name="dns_crypt_url_remove_success">DNSCrypt resolver deleted.</string>
 
-    <string name="dns_crypt_relay_remove_dialog_title">Delete DNSCrypt Relay?</string>
+    <string name="dns_crypt_relay_remove_dialog_title">Delete DNSCrypt relay?</string>
     <string name="dns_crypt_relay_remove_dialog_message">Delete the DNSCrypt relay from saved configuration.</string>
-    <string name="dns_crypt_relay_remove_success">DNSCrypt Relay deleted.</string>
+    <string name="dns_crypt_relay_remove_success">DNSCrypt relay deleted.</string>
 
-    <string name="dns_proxy_remove_dialog_title">Delete DNS Proxy?</string>
-    <string name="dns_proxy_remove_dialog_message">Delete the DNS Proxy from saved configuration.</string>
+    <string name="dns_proxy_remove_dialog_title">Delete DNS proxy?</string>
+    <string name="dns_proxy_remove_dialog_message">Delete the DNS proxy from saved configuration.</string>
     <string name="dns_proxy_remove_success">DNS Proxy deleted.</string>
 
     <string name="local_blocklist_download">Download Blocklists</string>
-    <string name="local_blocklist_download_desc">Download blocklists (around 60MB) to use this feature?</string>
+    <string name="local_blocklist_download_desc">Download blocklists (~60 MB) to use this feature?</string>
```

Comment:
```suggestion
    <string name="local_blocklist_download_desc">Download blocklists (around 60 MB) to use this feature?</string>
```
Since the other similar string is "Up to 350 MB"